src/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -105,213 +86,216 @@ private static void WriteTestDllAndName(TextWriter writer, string dll, string da
         /// <param name="level">Specifies the level of granuality to use for the output: either namepspace, class or function</param>
         /// <param name="writer">The StreamWriter which will be passed to WriteTestDllAndName() </param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", MessageId = "System.Reflection.Assembly.LoadFrom")]
```

Comment:
[Could] You could import System.Diagnostics.CodeAnalysis so you don't have to fully qualify System.Diagnostics.CodeAnalysis.SuppressMessage everywhere.

Reply:
Sure. we only have one place for that. But I will reference it as default library :)